### DISCUSSION TITLE
图表中英文标题间距如何缩小？

### DISCUSSION THREAD

**Discussion by SunicYosen** [sjtug/SJTUThesis#722] [提问]
使用bicaption生成的中英文双标题图表中，中英文标题间距如何缩小？

**Comment by PhotometricStereo** [comment]
一种可行的解决方法：实现了简单的双语标题间隔缩小功能:
`\bicaption{中文图题\vspace{-0.5cm}}{English figure title}`

在bicaption的第一个括号里面加`\vspace` 手工调整间隔。

- 调整前效果：
![图片](https://user-images.githubusercontent.com/101957989/185798239-013a46c2-c603-4d2d-8c06-98ed792337f2.png)
- 调整后效果
![图片](https://user-images.githubusercontent.com/101957989/185798167-4abc9969-20b6-4d5f-bb6c-e84e0cdbd899.png)

**Reply by PhotometricStereo** [reply]
发现一个副作用，会造成插图索引页的行距变窄！
![图片](https://user-images.githubusercontent.com/101957989/187700182-33b8edb6-364a-4e48-b823-aa4f7a3cf2cc.png)

**Comment by LogCreative** [comment]
在导言区添加
```latex
\DeclareBiCaptionSeparator{thinskip}{\vspace{-1ex}}
\captionsetup{bi-separator=thinskip}
```

可以查阅 [bicaption 的文档](https://mirrors.sjtug.sjtu.edu.cn/CTAN/macros/latex/contrib/caption/bicaption.pdf)。

> **Warning** 如果你的标题内容有多行，会导致行距不统一。

**Reply by SunicYosen** [reply]
在caption作者的帮助下问题得到解决，解决方案参见: [caption issue 133](https://gitlab.com/axelsommerfeldt/caption/-/issues/133).
